### ISSUE TITLE
Function "LOD" with "MANT_SIZE > 14"?

### ISSUE THREAD

**Issue by Oxygen-Chu** [dicecco1/fpga_cpfp#1]
I see the the function "LOD" can only be treated with "MANT_SIZE < 15".
So how about Single-Precision with "MANT_SIZE = 23", and even Double-Precision with "MANT_SIZE = 53"?

**Comment by dicecco1** [comment]
This library was specifically for low-precision floating-point operations. For single-precision or double-precision it'd be much better to use the built in support

**Comment by Oxygen-Chu** [comment]
Yes, but CPFP has a specific advantage over built-in float/double that the CPFP is portable, and not dependent upon Xilinx FP Core.
Will it be possible for you to have an updating for CPFP to support real arbitary precision floating-point synthesis?

Best Regards : -)
Oxygen

**Comment by dicecco1** [comment]
I'm not working on this project anymore unfortunately but it shouldn't be too much extra effort to augment for larger mantissa sizes
